common/source/docs/common-loading-firmware-onto-pixhawk.rst

```diff
@@ -47,36 +44,34 @@ Answer **Yes** when it asks you "Are you sure?".
 
    Mission Planner: Install FirmwareScreen
 
-Next it will try to detect which board you are using and it may ask you to unplug the board, press OK, and  plug it back in to detect the board type.
+Mission Planner will try to detect which board you are using. It may ask you to unplug the board, press OK, and plug it back in to detect the board type.
 
 .. figure:: ../../../images/Pixhawk_InstallFirmware2.png
    :target: ../_images/Pixhawk_InstallFirmware2.png
 
    Mission Planner: Install FirmwarePrompt
 
 
-If all goes well you will see some status appear on the bottom right
-including the words, "erase...", "program...", "verify.." and "Upload
-Done".  The firmware has been successfully uploaded to the board.
+If all goes well, you will see a status appear on the bottom right including the words: "erase...", "program...", "verify..", and "Upload Done". The firmware has been successfully uploaded to the board.
 
 It usually takes a few seconds for the bootloader to exit and enter the main code after programming or a power-up. Wait to press CONNECT until this occurs.
 
-.. note:: Updating the firmware to a newer version does not alter existing parameters unless the firmware is for a different vehicle, in which case parameters are reset to their default values for that vehicle. However, it is always a good idea to save your parameters to a file using the "Save to File" button on the Mission Planner's **CONFIG/Full Parameter Tree** tab before any firmware updates, just in case of any issues while updating.
+.. note:: Updating the firmware to a newer version does not alter existing parameters unless the firmware is for a different vehicle, in which case parameters are reset to their default values for that vehicle. However, it is always a good idea to save your parameters to a file using the "Save to File" button on the Mission Planner's **CONFIG/Full Parameter Tree** tab before any firmware updates, just in case of any issues while updating. Do not apply all parameters after upgrading to a new version as some parameters might have a different meaning.
```

Comment:
not actually certain this is true...we convert params that have change to another name, true but loading old ones wont actually load in that case....I do not think we change the meaning of any existing param on something that is released...perhaps rarely in DEV if we think no one really has used it yet...like on some quadplane frames I recall...so I do not think this statement should be included...you can upgrade firmware to any release version and re-apply old param files without unexpected consequences, to my knowledge

Reply:
Currently moving to Plane 4.1 and applying the old PID gains will cause issues because the parameter names are the same but the values have different meaning. There have been a few users on the forums that had this issue so documenting it would be useful before 4.1 leaves beta.

Reply:
old pid gains are different names...only the RLL2SRV_TCONST and PTCH2SRV_TCONST remained unchanged..which ones are you talking about?

Reply:
My bad on the PID parameters. I'm not sure where that came from in my head.

https://discuss.ardupilot.org/t/plane-4-1-0-beta/72434/67

Mainly sparked from this post by tridge and noticing that we do not have this recorded anywhere. It is important to tell users not to upload old parameters because of the possibility of similar parameters names even when the auto upgrade scripts works.

Reply:
something I do often...because I change firmware often when experimenting with code on the bench and reset defaults and then reload old param files...never have had an issue...will discuss more with Tridge...I am still not convinced that this is warranted...